tests/controller/insertcontent.js

```diff
@@ -610,6 +612,18 @@ describe( 'DataController', () => {
 				insertHelper( '<disallowedWidget>xxx</disallowedWidget>' );
 				expect( getData( doc ) ).to.equal( '<paragraph>f[]oo</paragraph>' );
 			} );
+
+			it( 'filters out disallowed attributes', () => {
```

Comment:
I'm also little surprised that it went so easy :) But looks like it really is. When disallowed element is a `Text` but in general `Text` is allowed in current position it means that this `Text` contains disallowed attributes and we have to remove them.